packages/core-utils/src/profile.js

```diff
@@ -27,8 +27,8 @@ export function filterProfileOptions(response) {
   return response;
 }
 
-function locationString(str, defaultStr) {
-  return str ? str.split(",")[0] : defaultStr;
+function locationString(str) {
+  return str ? str.split(",")[0] : null;
```

Comment:
`str?.split...`

Reply:
Refactored in 607cb0f2263907b88396ccd78574129c94d0b1a2